lib/service/service_test.go

```diff
@@ -495,6 +495,12 @@ func TestSetupProxyTLSConfig(t *testing.T) {
 				"h2",
 				"http/1.1",
 				"acme-tls/1",
+				"teleport-postgres-ping",
```

Comment:
Do we really need to duplicate every single entry here? Sounds like something that is very easy to miss when adding a new database protocol and very hard to detect. Could we, for example, generate all combinations by appending `-ping` on init for example?